woocommerce/woocommerce
# CYS - Update icon and text colors in the assembler

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the color of the WordPress icon to `gray-900` and also the color of the sections button to `gray-800`.

Closes https://github.com/woocommerce/woocommerce/issues/50385



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `wp-admin/admin.php?page=wc-admin&customizing=true&path=%2Fcustomize-store`.
2. Click `Customize your store` or `Start designing`.
3. When the assembler loads check the logo in the top left is black and the button colors are darker.

| Before | After |
|--------|--------|
| <img width="378" alt="Screenshot 2024-08-08 at 13 36 07" src="https://github.com/user-attachments/assets/548b11bb-a3e7-40b7-8998-24084011ab2c"> | <img width="379" alt="Screenshot 2024-08-08 at 13 35 46" src="https://github.com/user-attachments/assets/6af28775-047c-458f-8575-9abffdad9a98"> | 




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Update icon and text colors in the assembler.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
